azure_monitor/src/azure_monitor/auto_collection/__init__.py

```diff
@@ -19,18 +18,7 @@
 
 
 class AutoCollection:
-    def __init__(
-        self, meter: Meter, label_set: LabelSet, collection_interval: int = 60
-    ):
+    def __init__(self, meter: Meter, label_set: LabelSet):
         self._performance_metrics = PerformanceMetrics(meter, label_set)
         self._dependency_metrics = DependencyMetrics(meter, label_set)
         self._request_metrics = RequestMetrics(meter, label_set)
-
-        self._collect_task = PeriodicTask(
-            interval=collection_interval, function=self.collect
-        )
-
-    def collect(self):
-        self._performance_metrics.track()
```

Comment:
Are you adding some other way to collect the standard metrics?

Reply:
I suppose the observer is taking care of it now, right?

Reply:
Yes observer is taking care of this. As you have noticed, the observer metrics exporting shares the same exporting interval as the other metrics. We want the instrumentation experience to be easy for users, so adding another PushController is pretty bad (they must create another meter copy too). Ideally, we should do all of this in the wrapper work that we planned to do, as we are making this package an SDK rather than just the exporter.

For now, I will include a comment in the example file for your suggestion.